src/addons/transitions/__tests__/ReactTransitionGroup-test.js

```diff
@@ -128,7 +126,7 @@ describe('ReactTransitionGroup', function() {
       },
       render: function() {
         var children = [];
-        for (var i = 0; i < this.state.count; i++) {
+        for (let i = 0; i < this.state.count; i++) {
```

Comment:
It's been brought up multiple times that we shouldn't be running these tests internally anyways, so I'll look into that issue before we merge this.